docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -59,6 +119,50 @@ const compactHumanizeDate = (date) => {
     .replace(" year", "y");
 };
 
+/**
+ * Convert an Optional string of high, medium, low to a number of 0, 1, 2, 3
+ */
+const levelToIcon = (level) => {
+  switch (level?.toLowerCase()) {
+    case "high":
+      return 3;
+    case "medium":
+      return 2;
+    case "low":
+      return 1;
+    default:
+      return 0; // Default case for undefined or any other input
+  }
+};
+
+// COMPONENTS
+
+const IconRow = ({ iconComponent, number }) => {
+  const icons = Array.from({ length: number }, (_, index) => (
+    <div
+      key={index}
+      className={styles.iconWrapper}
+      style={{ zIndex: number - index }}
+    >
+      {iconComponent}
+    </div>
+  ));
+
+  return <div className={styles.iconRow}>{icons}</div>;
+};
```

Comment:
The result of this doesn't quite match up with the design for the repeated usage rate icon.

[Design](https://www.figma.com/design/3jfpae95Drbt19t9Q3nh0f/Connector-catalog-2.0?node-id=26-150&t=8Zx3QLAtumghbWBv-0):
<img width="815" alt="Screenshot 2024-06-05 at 3 36 02 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/b0a605f3-5b93-45b7-885d-be275543cac0">

This PR:
<img width="854" alt="Screenshot 2024-06-05 at 3 36 13 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/3565401d-b1b0-452b-91fb-13a805d74e73">

I think it might be better to just save the one- two- and three- variants of each icon as fully separate SVGs and use those directly instead of trying to build it with CSS, unless we can get a straightforward CSS solution for the repeated user icon that looks closer to the design.

Reply:
Ahh yeah. I had talked with @nora-airbyte and advocated for the CSS approach so we could have a reusable IconRow component since theyre common.

But Youre right, this is further off the designs than I wouldve liked.

Changing back

Reply:
@bnchrch Do you need svgs of the variants?

Reply:
Nope got them! Thanks to your lovely figma work :)